### PR TITLE
[RET-224] PO Feedback

### PR DESCRIPTION
**Pull request by jcmagsay**
### What's this PR do?
- Fix mobile positions of buttons for consistency
- Create mechanism for soft validation on Login page form
- Adding comments to Textfield component and Login page

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-224

### Who should review this PR?
@jcmagsay @mhfen

### CODE REVIEW

File: src/client/components/textfield.jsx [hunk 1]
```diff
@@ -117,6 +105,14 @@ export default class Textfield extends React.Component {
     }
   }
 
+  /**
+   * On keyup of the texfield input, change
+   * fieldValid flag to show proper validation
+   * states in the UI
+   *
+   * @method  blur
```
Comment (mhfen) [inline src/client/components/textfield.jsx]: `handleKeyUp`

File: src/client/pages/auth/login.jsx [hunk 2]
```diff
@@ -19,17 +26,61 @@ export default class Login extends Component {
...
+   * Handles submission of login form
+   * Will validate email and password fields
+   * based on their latest values
+   *
+   * @method  handleSubmit
+   */
   handleSubmit(e) {
     e.preventDefault();
+    this.setState({ 'emailError': !this.state.email.length });
+    this.setState({ 'passwordError': !this.state.password.length });
```
Comment (mhfen) [inline src/client/pages/auth/login.jsx]: Should this method be checking the state of each field for `isError`? I think we are doing something like that in GX.

Comment (jcmagsay) [inline src/client/pages/auth/login.jsx]: Yeah, I was trying to move away from the inline `ref` pattern with this method which will hopefully prevent multiple component updates, but I can change it to the GX way if you feel more comfortable with the `ref` pattern.

Comment (mhfen) [inline src/client/pages/auth/login.jsx]: I'm open to multiple methods of validating the form, as long as the submit action does not allow a submit if any of the fields is invalid. This looks like it just checks for presence.

Comment (jcmagsay) [inline src/client/pages/auth/login.jsx]: Good call, I put a placeholder in for now that will prevent the future `login` method from being called if the form is not valid.

**Pull request merged**